### PR TITLE
use snake_case for function naming

### PR DESCRIPTION
**Pull request by yennamsanthosh-skyflow**

### CODE REVIEW

File: samples/InsertSample.py [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+    global bearerToken
+    if is_expired(bearerToken):
+        bearerToken, _ = generate_bearer_token('<YOUR_CREDENTIALS_FILE_PATH>')
     return bearerToken
 
 
 try:
     config = Configuration(
-        '<YOUR_VAULT_ID>', '<YOUR_VAULT_URL>', tokenProvider)
+        '', '', token_provider)
```
Comment (skyflow-tejesh-reddy) [inline samples/InsertSample.py:17]: Should add back placeholders

File: samples/InsertSample.py [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
     client = Client(config)
 
     options = InsertOptions(True)
 
     data = {
         "records": [
             {
-                "table": "<TABLE_NAME>",
+                "table": "cards",
```
Comment (skyflow-tejesh-reddy) [inline samples/InsertSample.py:25]: Replace with placeholder in this entire file

File: samples/README.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ These are the samples for skyflow-python. In order to use these samples, you mus
 
 Below is a brief description of the samples:
 
-- [SATokenSample.py](https://github.com/skyflowapi/skyflow-python/blob/main/samples/SATokenSample.py): Contains a python program illustrating the usage of `ServiceAccount.generateBearerToken()`
+- [SATokenSample.py](https://github.com/skyflowapi/skyflow-python/blob/main/samples/SATokenSample.py): Contains a python program illustrating the usage of `ServiceAccount.generate_bearer_token()`
```
Comment (skyflow-tejesh-reddy) [inline samples/README.md:9]: Change to service_account

File: samples/README.md [hunk 4]
```diff
@@ -6,8 +6,8 @@ These are the samples for skyflow-python. In order to use these samples, you mus
...
 - [InsertSample.py](https://github.com/skyflowapi/skyflow-python/blob/main/samples/InsertSample.py) : Contains a python program illustrating the use of `Vault.Client.insert()`
 - [detokenizeSample.py](https://github.com/skyflowapi/skyflow-python/blob/main/samples/detokenizeSample.py): Contains a python program illustrating the use of `Vault.detokenize()`
-- [`getByIDSample.py`](https://github.com/skyflowapi/skyflow-python/blob/main/samples/getByIDSample.py): Contains a python program illustrating the use of `Vault.Client.getByID()`
-- [`invokeConnection.py`](https://github.com/skyflowapi/skyflow-python/blob/main/samples/invokeConnectionSample.py): Contains a python program illustrating the use of `Vault.Client.invokeConnection()`
```
Comment (skyflow-tejesh-reddy) [inline samples/README.md:13]: Change to smaller V in vault

File: samples/SATokenSample.py [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
 
 # cache token for reuse
 bearerToken = ''
 tokenType = ''
 
 
-def tokenProvider():
-    if isExpired(bearerToken):
-        bearerToken, tokenType = generateBearerToken(
+def token_provider():
```
Comment (skyflow-tejesh-reddy) [inline samples/SATokenSample.py:9]: global keyword fix in all samples

**Review by prathyusha-skyflow — APPROVED** [review]

**Review by skyflow-tejesh-reddy — APPROVED** [review]

**Pull request merged**